File: viewpagerdotsindicator/src/main/java/com/tbuonomo/viewpagerdotsindicator/DotsGradientDrawable.java
package com.tbuonomo.viewpagerdotsindicator;

import android.graphics.drawable.GradientDrawable;

public class DotsGradientDrawable extends GradientDrawable {

    private int currentColor;

    @Override
    public void setColor(int argb) {
        super.setColor(argb);

        currentColor = argb;
    }

    public int getCurrentColor() {
        return currentColor;
    }
}

File: viewpagerdotsindicator/src/main/java/com/tbuonomo/viewpagerdotsindicator/DotsGradientDrawable.kt
package com.tbuonomo.viewpagerdotsindicator

import android.graphics.drawable.GradientDrawable

class DotsGradientDrawable : GradientDrawable() {
    var currentColor: Int = 0
        private set

    override fun setColor(argb: Int) {
        super.setColor(argb)
        currentColor = argb
    }
}